src/Microsoft.ComponentDetection.Detectors/pnpm/Pnpm9Detector.cs

```diff
@@ -73,11 +73,12 @@ private void ProcessDependencyList(ISingleFileComponentRecorder singleFileCompon
         {
             var pnpmDependencyPath = this.pnpmParsingUtilities.ReconstructPnpmDependencyPath(name, dep.Version);
             var (component, package) = components[pnpmDependencyPath];
+            var (_, packageVersion) = this.pnpmParsingUtilities.ExtractNameAndVersionFromPnpmPackagePath(pnpmDependencyPath);
```

Comment:
we should bump the Pnpm Detector version

Reply:
done